header.php

```diff
@@ -1,15 +1,8 @@
 <?php
+    require "php/auth.php";
```

Comment:
auth.php only adds functions, so it shouldn't matter where it gets imported. The functions are used after the password import.